server/src/main/java/io/crate/metadata/view/ViewsMetadata.java

```diff
@@ -140,7 +140,8 @@ public static ViewsMetadata fromXContent(XContentParser parser) throws IOExcepti
                         ViewMetadata viewMetadata = new ViewMetadata(
                             stmt,
                             owner,
-                            searchPath == null ? SearchPath.pathWithPGCatalogAndDoc() : searchPath
+                            searchPath == null ? SearchPath.pathWithPGCatalogAndDoc() : searchPath,
+                            true
```

Comment:
Not sure about this one here. Is XContent only used for bwc? Did we move to binary serialization? Then this just defaults to the default setting value `true`.